site/content/docs/5.2/components/dropdowns.md

```diff
@@ -230,7 +230,12 @@ And putting it to use in a navbar:
 {{< example >}}
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
   <div class="container-fluid">
-    <a class="navbar-brand" href="#">Navbar</a>
+    <div class="navbar-brand me-auto me-lg-4">
+      <a class="stretched-link" href="#">
+        <img src="/docs/5.2/assets/brand/orange-logo.svg" width="50" height="50" alt="Boosted - Back to Home" loading="lazy">
```

Comment:
There is a discrepancy between what's inside Navbar examples and Global Headers examples regarding the `alt` information so I suggest to use this one (more like the navbar examples):

```suggestion
        <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" width="50" height="50" alt="Boosted" loading="lazy">
```

Anyway not in this PR we should align those `alt`s.